src/smtsolvers/LAScore.h

```diff
@@ -16,7 +16,8 @@ template<class EV>
 class LABestLitBuf {
 private:
     int size;
-    vec<std::pair < EV, Lit> > buf;
+    struct EVLitPair { EV first; Lit second; };
```

Comment:
Can we decide on one thing from `std::pair` vs `opensmt::pair` vs custom pair `struct`?

Reply:
Give more meaningful names to the components or just use `opensmt::pair`.

Reply:
Fixed to use opensmt::pair.